config/config.go

```diff
@@ -187,6 +188,21 @@ func (c WalkParams) ConfigureSNMP(g *gosnmp.GoSNMP) {
 	g.SecurityParameters = usm
 }
 
+type Filters struct {
+	Static  []StaticFilter  `yaml:"static,omitempty"`
+	Dynamic []DynamicFilter `yaml:"dynamic,omitempty"`
+}
+
+type StaticFilter struct {
+	Targets []string `yaml:"targets,omitempty"`
+	Indices []string `yaml:"indices,omitempty"`
```

Comment:
I see why this is a slice of string, but, maybe it would be worth doing this as a `[]uint` and doing bounds checking to make sure only a single byte worth of decimal integer is passed?

:thinking:

Reply:
I started with an int at the beginning then switch to string to have the concatenation part esier with the oid. 

Would you prefer to switch it to uint or simply try once to `strconv.Atoi` the value, in UnmarshalYAML for example ?

Reply:
I don't have a strong opinion here. But it would be good to have some kind of input validation.

Reply:
No strong opinion, either, just _some_ validation, as @SuperQ said.

Reply:
Implemented in `UnmarshalYAML` [below](https://github.com/prometheus/snmp_exporter/pull/624/files#diff-d2187742eb30a92ec406b45dcd849d5db523031d682af72683892e83dd6c5f0fR57)